sim/src/analytics.rs

```diff
@@ -256,7 +258,17 @@ impl Analytics {
                 self.problems_per_trip
                     .entry(trip)
                     .or_insert_with(Vec::new)
-                    .push((time, Problem::LargeIntersectionCrossing(t.parent)));
+                    .push((time, Problem::ComplexIntersectionCrossing(t.parent)));
+            }
+        }
+
+        if let Event::AgentEntersTraversable(a, Some(trip), Traversable::Turn(t), _) = ev {
+            let turn = map.get_t(t);
+            if a.to_type() == AgentType::Pedestrian && turn.is_crossing_arterial_intersection(map) {
+                self.problems_per_trip
```

Comment:
Good question! I'd only run it for one map. Let me run it for the whole shebang...

Reply:
Just for lakeslice is probably fine; it's the largest, so we'd see problems there if anywhere

And after we merge this PR, I'll regen this and upload to S3

Reply:
The total goes from 195MB to 245MB. lakeslice goes from 65MB to 79MB

I think this is fine (the data is a huge benefit), but it may soon be time to rethink some of the `Analytics` storage and compress it a bit

Reply:
Yikes, that is kind of a large growth.

> but it may soon be time to rethink some of the Analytics storage and compress it a bit

My first thought was putting something like `flate` into our `read_binary` `write_binary` chain, but I'm not sure how that would play with wasm. Probably not well... though apparently this exists: `https://github.com/drbh/wasm-flate`

Reply:
We already grab compressed files on the web:
https://github.com/a-b-street/abstreet/blob/2f71751929e26c1cc5a0ce405ea3ea1b06f90277/map_gui/src/load.rs#L275

Compression helps a bit, but the root problems with this file growing are,
1) We try to store lots of raw data per lane, intersection, and trip
2) Some of the stored types are extremely wasteful. Every `Duration` is an `f64`, even though we clamp the precision and could definitely get away with storing less. #317 is a very abandoned attempt to do that